src/noble/hashes.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2021 Cloudflare, Inc. and contributors.
+// Copyright (c) 2021 Cloudflare, Inc.
+// Licensed under the BSD-3-Clause license found in the LICENSE file or
+// at https://opensource.org/licenses/BSD-3-Clause
+
+import { sha256 } from '@noble/hashes/sha256'
+import { sha384, sha512 } from '@noble/hashes/sha512'
+import { CHash, wrapConstructor } from '@noble/hashes/utils'
+import { shake256 } from '@noble/hashes/sha3'
+import { HashID } from '../cryptoTypes.js'
+
+export const shake256_512 = wrapConstructor(() => shake256.create({ dkLen: 64 }))
```

Comment:
It requires this annotation to have the same type.
```suggestion
export const shake256_512 = wrapConstructor<Hash<Keccak>>(() => shake256.create({ dkLen: 64 }))

```